src/Apps/Artwork/Components/ArtworkLightbox.tsx

```diff
@@ -75,7 +84,7 @@ const ArtworkLightbox: React.FC<
 
   return (
     <>
-      {isDefault && (
+      {isDefault && shouldRenderFullImage && preloadImage && (
```

Comment:
I added a preloadImage && guard as well to avoid rendering a <link rel="preload"> with an undefined href